DAL/Repositories/PaymentRepository.cs

```diff
@@ -83,15 +86,6 @@ private async Task<IEnumerable<TransactionDTO>> TransactionSelector(int orderId,
             if (endDate != null) query = query.Where(a => endDate < a.TransactionTime);
 
             return query.ToList();
```

Comment:
According to name, this method should created selector and just responsible for composing a query but not execute a transaction.(query.ToList(); should be called somewhere else)

Reply:
i will change it